src/transport/SecureSessionMgr.cpp

```diff
@@ -194,8 +194,12 @@ CHIP_ERROR SecureSessionMgr::SendMessage(SecureSessionHandle session, PayloadHea
         *bufferRetainSlot = EncryptedPacketBufferHandle::MarkEncrypted(msgBuf.Retain());
     }
 
-    ChipLogProgress(Inet, "Sending msg from 0x" ChipLogFormatX64 " to 0x" ChipLogFormatX64 " at utc time: %" PRId64 " msec",
-                    ChipLogValueX64(localNodeId), ChipLogValueX64(state->GetPeerNodeId()), System::Layer::GetClock_MonotonicMS());
+    ChipLogProgress(Inet,
+                    "Sending message of type %d and protocolId %d with MsgId:%08" PRIX32 " on exchange %d from 0x" ChipLogFormatX64
+                    " to 0x" ChipLogFormatX64 " at utc time: %" PRId64 " msec",
+                    payloadHeader.GetMessageType(), payloadHeader.GetProtocolID(), packetHeader.GetMessageId(),
+                    payloadHeader.GetExchangeID(), ChipLogValueX64(localNodeId), ChipLogValueX64(state->GetPeerNodeId()),
```

Comment:
This is a very large amount of logging per message. It may impact performance on any embedded class device which has a simplistic logging interface with either blocking entries list, or if it has blocking semantics on full UART buffer or similar. Suggest making the default level of logging here configurable by application